tests/sample_outputs/run_attack_cnn_cola.txt

```diff
@@ -0,0 +1,59 @@
+Attack(
```

Comment:
Really minor thing, but when creating new tests, it's a good idea to add `/.*/` before "Attack" in line 1 (so like `/.*/Attack(` and another one between the printing of attack and `---Result 1---` (e.g. line 21). This is because sometimes, extra log output might be captured in these spots and cause test failures even though the main content are correct.